kcidb/db/abstract.py

```diff
@@ -1,7 +1,7 @@
 """Kernel CI reporting database - abstract database definitions"""
 
 from abc import ABC, abstractmethod
-import kcidb.orm as orm
+from kcidb.orm.query import Pattern
```

Comment:
Could you instead write:
```python
import kcidb.orm.query
```
And refer to `Pattern` as `kcidb.orm.query.Pattern`? We only have one reference here (so there's not much typing/reading overhead) and it would be good to be clear where it's coming from exactly.

Reply:
Oh!!! Okay. Done.